uploader.json

```diff
@@ -1426,20 +1426,6 @@
             ],
             "symbolic_name": "ceph_conf"
         },
-        {
-            "file": "/var/log/ceph/ceph.log",
-            "pattern": [
-                "[WRN] slow request"
-            ],
-            "symbolic_name": "ceph_log"
-        },
-        {
-            "file": "/var/log/ceph/()*ceph-osd.*\\.log$",
```

Comment:
@psachin leave these enteries here as `glob` entries are not supported in the `uploader.json` file.  Also this needs to match the spec in insights core which is `glob_file(r"var/log/ceph/ceph-osd*.log"`.  Which one is correct?

Reply:
Please follow the pattern above for this spec.